rita/src/rita_common/dashboard/network_endpoints.rs

```diff
@@ -38,3 +43,58 @@ pub fn set_settings(
 
     JsonStatusResponse::new(Ok("New settings applied".to_string()))
 }
+
+pub fn wipe(_req: HttpRequest) -> Result<HttpResponse, Error> {
+    if cfg!(not(debug_assertions)) {
```

Comment:
what does this do exactly?

Reply:
`debug_assertions` are true for `--debug` build and false for `--release`. Couldn't find better way than this. I'll add comment to this part.